CSS/introduction.css

```diff
@@ -17,26 +17,51 @@
 }
 
 .main-content {
-    max-width: 75%;
+    /* max-width: 75%; */
+    width: 70%;
+    height: max-content;
     margin: var(--xs);
-    border: 2px solid black;
```

Comment:
use variables instead of giving direct colors

Reply:
that border is for testing responsiveness.. i'll remove it after making components